iina/PlayerCore.swift

```diff
@@ -93,6 +93,12 @@ class PlayerCore: NSObject {
   }
   private var _touchBarSupport: Any?
 
+  /// `true` if this Mac is known to have a touch bar.
+  ///
+  /// - Note: This is set based on whether `AppKit` has called `MakeTouchBar`, therefore it can, for example, be `false` for
+  ///         a MacBook that has a touch bar if the touch bar is asleep because the Mac is in closed clamshell mode.
+  var hasTouchBar = false
```

Comment:
```suggestion
  var needsTouchBar = false
```

Based on the comment above.

Reply:
Latest commit renames the new property to needsTouchBar as recommended.